tests/src/com/unciv/dev/FasterUIDevelopment.kt

```diff
@@ -41,6 +41,7 @@ object FasterUIDevelopment {
             actor = "This could be your UI element in development!".toLabel()
         }
 
+        @Suppress("EmptyFunctionBlock")
         fun afterAdd() {
```

Comment:
same here

Reply:
This is a dev helper tool, this is legit